isis/src/base/objs/Pipeline/Pipeline_Darwin_x86_64_MacOSX10_13.truth

```diff
@@ -1,479 +0,0 @@
-Simple Pipe
```

Comment:
are we removing this because the generic Pipeline.truth file handles macOS now? (just checking)

Reply:
Yes, the new truth data should be the truth data for all OS's.